regulations/static/regulations/js/source/app-init.js

```diff
@@ -25,6 +26,7 @@ Backbone.$ = $;
     init: function() {
         Router.start();
         this.bindEvents();
+        $('table').stickyTableHeaders();
```

Comment:
Yes - you are correct.  If tables occur in the definitions - we would want to add the this.$el.find('table').stickyTableHeaders(); line in the SidebarView:loaded() callback as well.  I will double check but it appears that main-view and sidebar-view are the only two places where ajax content is inserted into the DOM.